ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.1.yaml

```diff
@@ -8,7 +8,7 @@ build_root:
     cluster: https://api.ci.openshift.org
     name: release
     namespace: openshift
-    tag: golang-1.10
+    tag: golang-1.12
```

Comment:
Should we really be changing the `4.1` file?

Reply:
Oh gosh great catch

Reply:
fixed now!